compliance/azure/ahub_manual/azure_ahub_utilization_with_manual_entry.pt

```diff
@@ -292,13 +292,44 @@ policy 'azure_ahub_policy' do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{data.summary_template}}"
     detail_template <<-EOS
 # Azure AHUB Utilization with Manual Entry
-| Subscription Name | Location | Resource Group | Instance Name | Instance Size | Image Publisher | Image Offer | Image Sku | Image Version |  Tags | Number Cores |
-| ------------- | -------------- | -------- | ------------- | --------------- | ----------- | --------- | ------------- |  ---- | ------------ |
-{{ range data.result -}}
-|{{.subscriptionName}} | {{.location}} | {{.resourceGroup}} | {{.name}} | {{.vmSize}} | {{.imagePublisher}} | {{.imageOffer}} | {{.imageSku}} | {{.imageVersion}} | {{.tags}} | {{.numberofcores}} |
-{{ end -}}
 ##### Note: {{data.note}}
   EOS
+    export jmes_path(data, 'result') do
```

Comment:
Note: We're changing this from:
```
export "result" do
```
to
```
export jmes_path(data, "result") do
```
After some discussion on phoenix team, for a couple reasons:
* Support for jq() instead of jmes_path() was added and needed a way to switch
* Future flexibility

Reply:
Err, after talking with Curt, I going to have it be the older format (text strings)... will update this.